modules/user/user.service.js

```diff
@@ -256,6 +256,14 @@ class UserService {
     return true;
   }
 
+
+  async switchUserStatus(id) {
+    const user = await this.getUserByFieldOrThrow('_id', id);
+
+    user.banned = !user.banned;
+
+    await user.save();
```

Comment:
If you have defined in graphql schema that returns type is boolean, then you need to return here `true` if everything is okay.

Reply:
fixed